infra/domains.tf

```diff
@@ -17,3 +17,7 @@ resource "digitalocean_domain" "thebrosmakeagame" {
 resource "digitalocean_domain" "daileybrue" {
   name = "daileybrue.coffee"
 }
+
+resource "digitalocean_domain" "whosbuying" {
+  name = "whosbuying.coffee"
+}
```

Comment:
This looks good! We also need to duplicate the three `digitalocean_record` resources in `infra/daileybrue.tf` and replace `daileybrue` with `whosbuying`.